src/material-experimental/mdc-slider/slider.ts

```diff
@@ -629,26 +628,26 @@ export class MatSlider extends _MatSliderMixinBase implements AfterViewInit, OnD
   }
 
   /** Gets the slider thumb input of the given thumb position. */
-  _getInput(thumbPosition: Thumb): MatSliderThumb {
+  _getInput(thumbPosition: Thumb = Thumb.END): MatSliderThumb {
```

Comment:
Default values for methods add a bit of code when transpiled to JS. Since these are all private methods, it's probably not worth the extra bytes for the convenience.